website/content/en/docs/upgrading-sdk-version/v0.1.0-migration-guide.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: v0.0.x to v0.1.0 
 linkTitle: v0.0.x to v0.1.0
-weight: 3
+weight: 2999982000
```

Comment:
This is basically the v0.1.0 migration guide, so let's apply the calculation based on that. I think it would be `999999000`.

```suggestion
weight: 999999000
```